rocAL/rocAL/include/decoders/image/fused_crop_decoder.h

```diff
@@ -83,4 +84,15 @@ class FusedCropTJDecoder : public Decoder {
     };
     bool _is_partial_decoder = true;
     std::vector <float> _bbox_coord;
+    std::mt19937 _rand_gen;
+    int64_t seed;
```

Comment:
All private variable should have _ before it.

Reply:
I think you are not using this variable. Check this.

Reply:
Done